tests/platform/cloud/AcceptanceTests_untrustedSettingsPut.js

```diff
@@ -0,0 +1,281 @@
+                }
+            }
+        }
+    }
+};
+
+gpii.tests.cloud.oauth2.untrustedSettingsPut.updatedPrefsSet = {
+    "contexts": {
+        "gpii-default": {
+            "name": "Default preferences",
+            "preferences": {
+                "http://registry.gpii.net/common/fontSize": 40,
+                "http://registry.gpii.net/common/backgroundColor": "black"
+            }
+        }
+    }
+};
+
+gpii.tests.cloud.oauth2.untrustedSettingsPut.expectedPrefsSet = {
+    "contexts": {
+        "gpii-default": {
+            "name": "Default preferences",
+            "preferences": {
+                "http://registry.gpii.net/common/fontSize": 40,
+                "http://registry.gpii.net/common/backgroundColor": "black",
+                "http://registry.gpii.net/common/foregroundColor": "white"
+            }
+        }
+    }
+};
+
+gpii.tests.cloud.oauth2.untrustedSettingsPut.testFilePath = gpii.tests.cloud.oauth2.untrustedSettingsPut.prefsDir + gpii.tests.cloud.oauth2.untrustedSettingsPut.key + ".json";
+
+gpii.tests.cloud.oauth2.untrustedSettingsPut.createTestFile = function () {
+    var initialPrefs = JSON.stringify(gpii.tests.cloud.oauth2.untrustedSettingsPut.initialPrefsSet);
+    fs.writeFile(gpii.tests.cloud.oauth2.untrustedSettingsPut.testFilePath, initialPrefs);
+};
+
+gpii.tests.cloud.oauth2.untrustedSettingsPut.cleanUpTestFile = function () {
+    fs.unlinkSync(gpii.tests.cloud.oauth2.untrustedSettingsPut.testFilePath);
+};
+
+gpii.tests.cloud.oauth2.untrustedSettingsPut.verifyUpdateResponse = function (responseText, expectedKey, expectedMsg, expectedPrefsSet) {
+    var response = JSON.parse(responseText);
+    jqUnit.assertEquals("The returned key in the response is correct", expectedKey, response.userToken);
+    jqUnit.assertDeepEq("The returned message set in the response is correct", expectedMsg, response.message);
+
+    var filePath = gpii.tests.cloud.oauth2.untrustedSettingsPut.prefsDir + expectedKey + ".json";
+    var savedPrefs = require(filePath);
+    jqUnit.assertDeepEq("Saved preferences are correct", expectedPrefsSet, savedPrefs.flat);
+};
+
+fluid.defaults("gpii.tests.cloud.oauth2.untrustedSettingsPut.requests", {
+    gradeNames: ["fluid.component"],
+    components: {
+        untrustedSettingsPutRequest: {
+            type: "kettle.test.request.http",
+            options: {
+                path: "/%key/untrusted-settings",
+                port: 8081,
+                method: "PUT",
+                termMap: {
+                    key: "{testCaseHolder}.options.key"
+                }
+            }
+        }
+    }
+});
+
+// For successful workflows that update user settings from /untrusted-settings endpoint
+// using access tokens granted by /access_token endpoint
+gpii.tests.cloud.oauth2.untrustedSettingsPut.mainSequence = [
+    {
+        funcName: "gpii.tests.cloud.oauth2.untrustedSettingsPut.createTestFile"
+    },
+    { // 0
```

Comment:
With the insertion of the `createTestFile` task, the numbering is now off-by-one.

We can update the numbering, or just remove it (as nobody else is using the `mainSequence` any more).